src/services/locales.service.ts

```diff
@@ -0,0 +1,46 @@
+import intl from 'react-intl-universal'
+
+type Dictionary = { [label: string]: string }
+
+export interface LocaleData {
+  lang: string
+  locales: Dictionary
+}
+
+export class Locales {
```

Comment:
A bit strange for me why you did not place it in the UI section.

Reply:
The UI section is for UI pages only. This class is a part of service worker, that's why I put it here.